packages/opentelemetry-instrumentation/src/platform/node/types.ts

```diff
@@ -14,6 +14,15 @@
  * limitations under the License.
  */
 
+import { Instrumentation } from '../../types';
+
+export interface InstrumentationBaseNode<T = any> extends Instrumentation {
+  loadInstrumentation(instrumentationModuleDefinitions: InstrumentationModuleDefinition<T>
+    | InstrumentationModuleDefinition<T>[]
+    | void
```

Comment:
why `| void`?

Reply:
browser

Reply:
But this is in file platform/node/types.ts